### ISSUE TITLE
Unable to run Jasmine tests with fixtures

### ISSUE THREAD

**Issue by almagest** [searls/jasmine-rails#123]
Hi,

I'm attempting to use jasmine-rails to process coffeescript (both code & tests), but I keep seeing "ReferenceError: Can't find variable: Regions in [...]/tmp/jasmine/assets/jasmine-specs.js (line 10723)". I check the compiled jasmine-specs.js file and I see Regions defined before line 10723. Can you offer any suggestions on how to get these tests running? Also, let me know if you need to see anything else. Thanks.

Relevant gems in use:
- jasmine (1.3.2)
- jasmine-core (1.3.1)
- jasmine-rails (0.7.1)

Includes:
- jasmine-jquery.js

Failing test (settings_spec.js.coffee):

``` coffeescript
describe 'Settings', ->
  beforeEach ->
    loadFixtures('settings.html')
    Regions.setup()
    Modal.setup()
    return

  it 'should disable all locations dropdowns except APAC on page load', ->
    expect($("#locations_apac")).toBeHidden()
    expect($("#locations_emea")).toBeVisible()
```

**Comment by guilhermepontes** [comment]
Thank you @nathany! Helped me a lot! :heart: :grin:

**Comment by nathany** [comment]
You’re welcome. ;-)

**Comment by greshny** [comment]
@nathany thank you very much!!!! :+1:

**Comment by joker-777** [comment]
Please have a look to my PR https://github.com/travisjeffery/jasmine-jquery-rails/pull/12. It will add jasmine-jquery fixture support for Rails without special configs necessary and it even supports HAML!

**Comment by ccallebs** [comment]
It seems like I can only choose to run the specs on the command line or via `http://localhost:3000/specs` using the solution above.

``` javascript
jasmine.getFixtures().fixturesPath = "../../spec/javascripts/fixtures";
```

It would be nice to conditionally load the fixtures based on whether it was via the browser or the command line. I'll look into that.

**Comment by enkanba** [comment]
Hi, @ccallebs, were you able to figure out how to make it run for both command line and via browser?
I am running into the same problem.

**Comment by ccallebs** [comment]
@enkanba I never was -- we just settled on the command line. Sorry :(

**Comment by searls** [comment]
(Maybe) fixed by #216
